docker-compose.yml

```diff
@@ -166,7 +196,7 @@ services:
   #
   faf-java-api:
     container_name: faf-java-api
-    image: faforever/faf-java-api:v2.2.8
+    image: faforever/faf-java-api:v2.2.7
```

Comment:
why?

Reply:
Actually I did rebase and looks like that was the outcome xD. Will revert.